RELEASE.md

```diff
@@ -151,5 +151,7 @@ appearing.  To check for progress:
 [Crates.io agave-validator](https://crates.io/crates/agave-validator) should have an updated agave-validator version.  This can take 2-3 hours, and sometimes fails in the `agave-secondary` job.
 If this happens and the error is non-fatal, click "Retry" on the "publish crate" job
 
+<!-- TODO: fix this dead link to actual 
 ### Update software on testnet.solana.com
 See the documentation at https://github.com/solana-labs/cluster-ops/. devnet.solana.com and mainnet-beta.solana.com run stable releases that have been tested on testnet. Do not update devnet or mainnet-beta with a beta release.
```

Comment:
We don't have any public documentation that replaces cluster-ops. I suggest removing this sentence:

> See the documentation at https://github.com/solana-labs/cluster-ops/.